### PR TITLE
[admin] ansible script for the nano, renamed imu_channel, updated DriveVelData in a couple code bases

### PR DESCRIPTION
**Pull request by cgiger00**

### CODE REVIEW

**Comment by cgiger00** [comment]
changed a couple things that affect other ppl's code bases, just checking to make sure they don't mess w things

File: ansible/roles/jetson_networks/files/can_ip_link.sh [hunk 1]
```diff
@@ -2,5 +2,5 @@
 ip link set can0 type can bitrate 1000000
 ip link set can0 txqueuelen 1000
 ip link set up can0
-nvpmodel -m 0
+nvpmodel -m 2
```
Comment (qhdwight) [inline ansible/roles/jetson_networks/files/can_ip_link.sh:5]: Nice

File: jetson/nav/main.cpp [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #include <memory>
 #include <fstream>
 #include <iostream>
-#include <filesystem>
+// #include <filesystem> does not compile with ubuntu18
```
Comment (qhdwight) [inline jetson/nav/main.cpp:5]: :+1:

**Comment by rbridges12** [comment]
Did you actually change the name of imu_channel to imu? when I check out this PR, the folder is still called jetson/imu_channel. Or am I just being dumb

**Review by rbridges12 — APPROVED** [review]
looks good now

**Review by qhdwight — APPROVED** [review]

**Pull request merged**